framework/src/org/checkerframework/qualframework/base/QualifiedTypeFactoryAdapter.java

```diff
@@ -65,10 +65,8 @@ void doPostInit() {
     }
 
     @Override
-    protected QualifierDefaults createQualifierDefaults() {
-        QualifierDefaults result = super.createQualifierDefaults();
-        getCheckerAdapter().setupDefaults(result);
-        return result;
+    protected void addCheckedCodeDefaults(QualifierDefaults defs) {
+        getCheckerAdapter().setupDefaults(defs);
```

Comment:
This class is the shim between the qual type factory and the annotated type factory.  Most of these methods don't call super, but don't state why.  (It's because the super behavior would not be correct for the qual type factory.)  Adding a comment to just this method would be confusing -- as in why is this method different than the other methods.